assets/libraries/cloudformation.rego

```diff
@@ -99,9 +99,13 @@ get_resource_accessibility(nameRef, type, key) = info {
 
 	keys := policy.Properties[key]
 
-	get_name(keys[_]) == nameRef
+	get_name(keys) == nameRef
```

Comment:
I think we should keep` get_name(keys[_]) == nameRef`, which it will require more conditions